src/isar_robot/robotinterface.py

```diff
@@ -175,3 +176,6 @@ def pause(self) -> None:
 
     def resume(self) -> None:
         return
+
+    def generate_media_config(self) -> MediaConfig:
+        return
```

Comment:
Should we return a default `MediaConfig` instance to match the function type-hinting?

Reply:
Isn't it better to send None than som random value? It makes it easier for isar  to know that there is no valid mediaConfig

Reply:
Raising an exception could be a good idea!

Reply:
Will change return type to optional both here and in isar